### DISCUSSION TITLE
Custom build of rustdesk

### DISCUSSION THREAD

**Discussion by kempfsolutions** [rustdesk/rustdesk#4350] [General]
### Bug Description

If someone already had install the rustdesk client, it do not uses the configuration values of the customized client.

### How to Reproduce

1) Fork rustdesk
2) Set Variables RENDEZVOUS_SERVER & RS_PUB_KEY 
3) Build
4) Run rustdesk 

### Expected Behavior

If the client is build with these settings, they should be preferred. 

### Operating system(s) on local side and remote side

any

### RustDesk Version(s) on local side and remote side

1.2.0 => 1.2.0

### Screenshots

No screenshots.

**Comment by SergeyMy** [comment]
rustdesk is both a client and a server and a service
maybe you have a service running? (settings vary)
try **sudo rustdesk**

**Comment by kempfsolutions** [comment]
The first time you start the rustdesk client, it add configuration data to the AppData directory (Windows).
If you start the a new one or a different executeable in my case of the fork, it do not use the settings from the build, it uses the settings of this directory of that other version.

**Comment by SergeyMy** [comment]
in version 1.2.0+, the parameters format in the settings file has recently changed
